src/AppCommon/Infra/SharedOptions.cs

```diff
@@ -27,12 +27,20 @@ class SharedOptions
         Arity = ArgumentArity.ZeroOrOne
     };
 
+    public static readonly Option<bool> skipThroughputCollection =
+        new("--skipThroughputCollection")
+        {
+            Description = "It returns only the queue names with zero throughput.",
+            Arity = ArgumentArity.ZeroOrOne
+        };
+
     public static readonly Option<int> runtimeInHours =
         new(name: "--runtime", getDefaultValue: () => 24) { IsHidden = true };
 
     public string CustomerName { get; set; }
     public bool RunUnattended { get; private set; }
     public bool SkipVersionCheck { get; private set; }
+    public bool SkipThroughputCollection { get; private set; }
```

Comment:
Does this appear in the report anywhere? Should it?

We should be able to differentiate between someone running the tool with this flag enabled vs one with zero throughput.

Reply:
Well, I guess the account should have details about what kind of license they are on, hence I don't think we need to use the report to tell us what kind of customer we are dealing with.
So for normal licenses, if we happen to get a report without any throughput today, my guess is that the account manager would be questioning it and asking for clarification.

Reply:
@mikeminutillo before I merge, just want to confirm my logic for not needing this extra data on the report makes sense?

Reply:
I think this is OK, as long as the account managers think to look for it. It should be pretty obvious from the data collected.